src/frontend-scripts/components/section-main/Profile.jsx

```diff
@@ -13,6 +13,20 @@ const mapDispatchToProps = dispatch => ({
 	fetchReplay: gameId => dispatch(fetchReplay(gameId))
 });
 
+// TODO: I'd like to make this reusable as it's repeated in UserPopup.
+// As I'm not too familiar with the codebase as of yet, I just duplicated the logic.
```

Comment:
You can put this function in one of the utility files and just import it as necessary

Reply:
alright, I just wasn't sure which utility file to stick itin.

Reply:
addressed PR comments!